src/pages/Login/LoginPage.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { Alert, Row, Col, Form, FormGroup, FormControl, Icon, Button, HelpBlock } from 'patternfly-react';
+import { connect } from 'react-redux';
+import * as UserAction from '../../actions/UserAction';
+import PropTypes from 'prop-types';
+import { KEY_CODES } from '../../config';
+
+const kialiTitle = require('../../assets/img/kiali-title.svg');
+
+type LoginProps = {
+  dispatch: any;
+};
+
+type LoginState = {
+  username: string;
+  password: string;
```

Comment:
I wonder how storing password in state and/or redux could be vulnerable to some XSS attack. Shouldn't we store a cookie instead? Or is it planned for a future task?